mindjunkies/lecture/management/commands/populate_lecture.py

```
@@ -0,0 +1,86 @@
+        LastVisitedModule.objects.all().delete()
+
+        # Get existing courses, modules, and users
+        courses = Course.objects.all()
+        modules = Module.objects.all()
+        students = User.objects.filter(is_teacher=False)
+
+        if not courses.exists() or not modules.exists():
+            self.stdout.write(self.style.ERROR("No courses or modules found. Please populate courses first."))
+            return
+        if not students.exists():
+            self.stdout.write(self.style.WARNING("No students found. Lecture completions and last visited modules will not be created."))
```

Ruff (0.8.2)

Line too long (137 > 119)

(E501)